test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -1381,7 +1387,11 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 					klog.Errorf("failed to get events for pod %s/%s; error: %v", pod.Namespace, pod.Name, err)
 				}
 				for _, e := range events {
-					if e.Reason == "FailedScheduling" && strings.Contains(e.Message, errMsg) {
+					ok, err := regexp.MatchString(errMsg, e.Message)
```

Comment:
why regexp and not just `strings.Contain`?

Reply:
because of https://github.com/openshift-kni/numaresources-operator/pull/942/files/d597b510435aeae0075b404465c6ad0b923dcbee..e6ba8e101af0232dd83823ad3e54a4249bf019db#diff-3960361d793f8bc715bd7e1caaf3cd3d976577c8598ee636f4d268a6dac7165dR1527 
That asked, I think I should have put that in a different commit with more clarification. I'll consider that next time.

Reply:
ok, no big deal then.